### PR TITLE
Remember when fsck was last run and remind after 90d

### PR DESCRIPTION
**Pull request by dominikschulz**
This adds a reminder when fsck was not run within the last 90d (or
never).

RELEASE_NOTES=[ENHANCEMENT] Remind to run gopass fsck after 90d

Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

**Review by AnomalRoil — CHANGES_REQUESTED** [review]
Let's have an extra case in `printReminder` that would allow us to disable this feature through an env var, shall we?

**Comment by dominikschulz** [comment]
Yes, sounds good. I was worried that some people might feel annoyed, too.

File: internal/action/reminder.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	if !ctxutil.IsTerminal(ctx) {
+		return
+	}
+	if sv := os.Getenv("GOPASS_NO_REMINDER"); sv != "" {
+		return
+	}
+
+	if s.rem.Overdue("fsck") {
+		out.Notice(ctx, "You haven't run 'gopass fsck' in a while.")
+		return
```
Comment (AnomalRoil) [inline internal/action/reminder.go:24]: Note that a return statement here means that we will warn only about fsck even if we should also warn about updates and audit.

People might ignore `fsck` but be more interested in the `update` one I guess.

**Review by AnomalRoil — DISMISSED** [review]

**Comment by AnomalRoil** [comment]
Please solve the conflicts too :)

File: internal/action/reminder.go (hunk 1 above)
Comment (dominikschulz) [reply internal/action/reminder.go]: Actually this is intentional! I did only want to print one notice per day.
Added a comment and put update first.

**Review by AnomalRoil — APPROVED** [review]

**Pull request merged**